<?xml version="1.0"?>

<launch>
    <node pkg="bot_sim" type="dstarlite" name="dstarlite" output="screen">
        <param name="map_topic_name" value="/map" />
        <param name="map_frame_name" value="map" />
        <param name="robot_frame_name" value="virtual_frame" />
        <param name="dynamic_map_topic_name" value="/grid" />
        <param name="goal_topic_name" value="/clicked_point" />
    </node>

</launch>
